src/lib/upload-forms.js

```diff
@@ -56,6 +57,7 @@ module.exports = (projectDir, subDirectory, options) => {
       doc._attachments.xml = attachmentFromFile(xformPath);
 
       return promiseChain
+        .then(() => warnUploadOverwrite.preUploadByXml(db, doc._id, xml))
```

Comment:
I take it this will warn every time someone changes a form intentionally? Could also have a `postUploadByXml` to store the last known uploaded version?